examples/basic/package.json

```diff
@@ -7,15 +7,15 @@
     "start": "remix-serve build"
   },
   "dependencies": {
-    "@remix-run/node": "1.6.4",
```

Comment:
@mcansh This should have been at `1.7.0` btw, so it seems like the version script didn't run for some reason 🤔

Reply:
yeah changesets does the bumping now, but before we would always use `scripts/version.js` which would bump the examples